### PR TITLE
Bug/2856 old all traffic

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #2856 

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/analytics/components/dashboard-details/AnalyticsDashboardDetailsWidgetTopAcquisitionSources.js [hunk 1]
```diff
@@ -20,71 +20,33 @@
...
-									mdc-layout-grid__cell--span-4-phone
-								">
-								<LegacyAnalyticsDashboardWidgetTopAcquisitionSources />
-							</div>
-						</div>
-					</div>
+			</Cell>
+			<Cell size={ 12 }>
+				<Layout>
+					<DashboardAllTrafficWidget { ...widgetComponentProps } />
```
Comment (aaemnnosttv) [inline assets/js/modules/analytics/components/dashboard-details/AnalyticsDashboardDetailsWidgetTopAcquisitionSources.js:47]: @eugene-manuilov Should we maybe use [`LegacyDashboardAllTraffic`](https://github.com/google/site-kit-wp/blob/404d20abe2685203fa71e9ef3af7b7ec1c4e8c91/assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/LegacyDashboardAllTraffic.js) here instead?

In which case, I think we may even be able to use that in place of `AnalyticsDashboardDetailsWidgetTopAcquisitionSources` altogether?

File: assets/js/modules/analytics/components/dashboard-details/AnalyticsDashboardDetailsWidgetTopAcquisitionSources.js [hunk 2]
```diff
@@ -20,71 +20,33 @@
...
 
 	return (
 		<Fragment>
-			<div className="
-					mdc-layout-grid__cell
-					mdc-layout-grid__cell--span-12
-				">
+			<Cell size={ 12 }>
 				<DashboardModuleHeader
 					title={ __( 'All Traffic', 'google-site-kit' ) }
```
Comment (felixarntz) [inline assets/js/modules/analytics/components/dashboard-details/AnalyticsDashboardDetailsWidgetTopAcquisitionSources.js:41]: This should also say "Your Traffic at a Glance" instead.

**Review by felixarntz — CHANGES_REQUESTED** [review]
LGTM, just one more thing to change here.

I'll quickly do that, and I'll also take care of removing some of these components, which we shouldn't need anymore.

File: assets/js/modules/analytics/components/dashboard-details/AnalyticsDashboardDetailsWidgetTopAcquisitionSources.js (hunk 1 above)
Comment (felixarntz) [reply assets/js/modules/analytics/components/dashboard-details/AnalyticsDashboardDetailsWidgetTopAcquisitionSources.js]: @aaemnnosttv @eugene-manuilov I went ahead and deleted this and other now unused components now, I think keeping it in the component where it used to be is fine, even though naming-wise this is a bit worse. But it's soon-to-be-removed legacy code anyway.

**Review by felixarntz — APPROVED** [review]
Great!

**Pull request merged**